clippy_lints/src/methods/mod.rs

```diff
@@ -3162,6 +3163,32 @@ declare_clippy_lint! {
     "collecting an iterator when collect is not needed"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    ///
+    /// Checks for `Command::arg()` invocations that look like they
+    /// should be multiple arguments instead, such as `arg("-t ext2")`.
+    ///
+    /// ### Why is this bad?
+    ///
+    /// Arguments are not split by space. An argument like `arg("-t ext2")`
```

Comment:
nit: worth making it clear that the API is the one doing the not-splitting

```suggestion
    /// `Command::arg()` does not split arguments by space. An argument like `arg("-t ext2")`
```